io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -134,6 +134,12 @@
 	public static final String PARAM_OMIT_NIL_REASON = "xml.notNil.omitNilReason";
 
 	/**
+	 * The parameter name for the flag specifying if the output of geometry
+	 * coordinates should be formatted.
+	 */
+	public static final String PARAM_GEOMETRY_FORMAT = "geometry.write.format";
```

Comment:
As `format` could be anything I would vote for chosing a name that makes it easier to identify what the settings is about. Maybe `decimalFormat` instead of `format`?